library/std/src/sys/unix/ext/process.rs

```diff
@@ -171,8 +171,32 @@ pub trait ExitStatusExt {
     fn from_raw(raw: i32) -> Self;
 
     /// If the process was terminated by a signal, returns that signal.
+    ///
+    /// Ie, if `WIFSIGNALED`, this returns `WTERMSIG`.
```

Comment:
I think `i.e.` is not clear for non-native speakers (and maybe some native speakers).
```suggestion
    /// In other words, if `WIFSIGNALED`, this returns `WTERMSIG`.
```